README.md

```diff
@@ -129,8 +130,31 @@ config.middleware.use(IPinfoMiddleware, {
   http_client: "",
   countries: "",
   filter: "",
+  ipselector: XForwardedIPSelector,
 })
 ```
+### IP Selection Mechanism
+
+By default, the IP is used by ignoring the reverse proxies depending on whether we are behind a reverse proxy or not.
+
+Since the desired IP by your system may be in other locations, the IP selection mechanism is configurable and some alternative built-in options are available.
+Using built-in IP selectors
```

Comment:
```suggestion
Since the actual desired IP may be something else, the IP selection mechanism is configurable.

Here are some built-in mechanisms:
```